### PR TITLE
Add tests data

### PR DESCRIPTION
**Pull request by benjamin-sommer**

### CODE REVIEW

**Comment by ebwinters** [comment]
will check and add unit tests tomorrow

File: funcs/getWorldometers.js [hunk 1]
```diff
@@ -97,6 +99,18 @@ function fillResult(html, yesterday = false) {
...
+		// get tests
+		if (i % totalColumns === testsColIndex) {
+			result[result.length - 1].tests = getCellData(cell);
+		}
+
+		// get total tests per one million population
+		if (i % totalColumns === testsPerOneMillionColIndex) {
+			const testsPerOneMillion = cell.children.length !== 0 ? cell.children[0].data : '';
+			result[result.length - 1].testsPerOneMillion = parseFloat(testsPerOneMillion.split(',').join(''));
+			result[result.length - 1].updated = Date.now();
```
Comment (EliteDaMyth) [inline funcs/getWorldometers.js:112]: There is already an updated thing on line 100.

**Comment by puf17640** [comment]
You missed the US state scraping in /funcs/getStates.js

**Comment by benjamin-sommer** [comment]
> You missed the US state scraping in /funcs/getStates.js
> https://www.worldometers.info/coronavirus/country/us/

@puf17640 Can you review?

**Comment by ebwinters** [comment]
writing tests then will merge

**Pull request merged**